Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/ActiveEvent.cs

```diff
@@ -61,6 +61,12 @@ public GameObject AudioEmitter
             private set;
         }
 
+        public string MessageToSend
```

Comment:
Performance wise, UnityEvent are slower than delegates but they can nicely be set straight from the inspector so I find them very powerful.
It's also cool how you create your own dynamic events (for ex. call this function that accept a Color), so to me it's definitely the way to go.
To finish we all know how slow SendMessage is ! ;-)